llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -32,6 +32,7 @@
 #include "llvm/IR/LLVMContext.h"
 #include "llvm/IR/LegacyPassManager.h"
```

Comment:
Do we still need this include?

Reply:
Ah..good catch. I do not think we need this include anymore. Thanks. I will remove it in the upcoming commit.